src/index.js

```diff
@@ -0,0 +1,6 @@
+import styles from './assets/scss/app.scss';
+import {App} from "./app";
```

Comment:
```suggestion
import { App } from './app';
```

use space inside brace and use single quote